.gitmodules

```diff
@@ -1,4 +1,24 @@
 [submodule "hersan"]
 	path = hersan
 	url = git@git.vauxoo.com:vauxoo/hersan.git
-	branch = 12.0
+	branch = 14.0
+[submodule "mexico"]
+	path = mexico
+	url = git@git.vauxoo.com:vauxoo/mexico.git
+	branch = 14.0
+[submodule "hr"]
+	path = hr
+	url = git@github.com:Vauxoo/hr.git
+	branch = 14.0
+[submodule "hr-advanced"]
+	path = hr-advanced
+	url = git@git.vauxoo.com:vauxoo/hr-advanced.git
+	branch = 14.0
+[submodule "server-tools"]
```

Comment:
Is server-tools really required?

I ask because, if it's only to install `company_country`, that module is deprecated since 13.0

Reply:
I added it because Vx/mexico has it 
https://git.vauxoo.com/vauxoo/mexico/-/blob/14.0/oca_dependencies.txt#L2

Reply:
you right, it's not need it